src/components/WizardPattern/WizardPattern.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
```

Comment:
Suggest use of enzyme mount here checkout the [about modal test](https://github.com/patternfly/patternfly-react/blob/master/src/components/AboutModal/AboutModal.test.js)

Reply:
I think I will switch all these tests over to `enzyme` soon anyway in order to add shallow-rendered method tests.